x-pack/plugins/fleet/server/routes/setup/handlers.ts

```diff
@@ -55,18 +47,12 @@ export const getFleetStatusHandler: RequestHandler = async (context, request, re
   }
 };
 
-export const createFleetSetupHandler: RequestHandler<
-  undefined,
-  undefined,
-  TypeOf<typeof PostFleetSetupRequestSchema.body>
-> = async (context, request, response) => {
+export const fleetSetupHandler: RequestHandler = async (context, request, response) => {
   try {
     const soClient = context.core.savedObjects.client;
     const esClient = context.core.elasticsearch.client.asCurrentUser;
-    const body = await setupIngestManager(soClient, esClient);
-    await setupFleet(soClient, esClient, {
-      forceRecreate: request.body?.forceRecreate ?? false,
-    });
+    const body: PostIngestSetupResponse = { isInitialized: true };
```

Comment:
This causes a regression: https://github.com/elastic/kibana/issues/97401

Reply:
Thanks for catching it